code-coverage.sh

```diff
@@ -1,3 +1,4 @@
+cd stellar_rust_sdk
 CARGO_INCREMENTAL=0 RUSTFLAGS='-Cinstrument-coverage' LLVM_PROFILE_FILE='cargo-test-%p-%m.profraw' cargo test
```

Comment:
```suggestion
CARGO_INCREMENTAL=0 RUSTFLAGS='-Cinstrument-coverage' LLVM_PROFILE_FILE='cargo-test-%p-%m.profraw' cargo test --manifest-path=stellar_rust_sdk/Cargo.toml
```
Dit is ook in lijn met de github workflows voor dit project.